src/util.js

```diff
@@ -174,3 +176,17 @@ export function compare_versions(a, b) {
 export function image_name(image) {
     return image.RepoTags ? image.RepoTags[0] : "<none>:<none>";
 }
+
+// Taken from cockpit `pkg/storaged/utilsx.jsx`
+export function fmt_to_fragments(fmt) {
+    const args = Array.prototype.slice.call(arguments, 1);
+
+    function replace(part) {
+        if (part[0] == "$") {
+            return args[parseInt(part.slice(1))];
+        } else
+            return part;
+    }
+
+    return React.createElement.apply(null, [React.Fragment, { }].concat(fmt.split(/(\$[0-9]+)/g).map(replace)));
+}
```

Comment:
This is already in src/lib with this commit https://github.com/cockpit-project/cockpit-podman/commit/47a97efebdd036b0f45f2a7388beea8dfbaac7cd

Reply:
Ooops, sorry, fixed